PutuWeek2

maranthadawkins opened the pull request:
my post

Commit:
Merge remote-tracking branch 'eddymankim/master'

evan-erdos committed:
added gitignore from personal website

Commit:
Week 2 Post

Commit:
Putu Week 2 Fixing Thumbnail

evan-erdos commented:
looks mostly OK, a few things:
- change `PutuWeek2.markdown` to `yourAndrewID-2.md` to match the naming convention
- remove temporary files or pull again to get the `.gitignore`, namely delete `_site` and `.sass-cache`

Commit:
Putu Editing Week 2 Post

Commit:
Merge remote-tracking branch 'eddymankim/development'

Commit:
Merge remote-tracking branch 'eddymankim/master'

Commit:
?

Commit:
?

# Conflicts:
#	_site/README.md
#	_site/feed.xml
#	_site/index.html
#	_site/test-project.html

evan-erdos commented:
try again, and it should be fine
